pkg/graph/resolvers/it_gov_task_statuses.go

```diff
@@ -161,6 +161,21 @@ func GrbMeetingStatus(intake *models.SystemIntake) (models.ITGovGRBStatus, error
 		if intake.Step == models.SystemIntakeStepGRBMEETING { //if the step is GRB meeting, status is awaiting decision
 			return models.ITGGRBSAwaitingDecision, nil
 		}
+
+		if intake.GrbReviewType == models.SystemIntakeGRBReviewTypeAsync {
```

Comment:
Thanks Tom, I took a look at the logic here, it looks like it matches the condition from the JIRA ticket.